LevelUp.Pos.ProposedOrders.Tests/SplitTenderTests.cs

```diff
@@ -68,23 +69,25 @@ public void SplitTenderExample_LevelUp_Then_Cash()
             // tax amount unchanged
             // spend amount unchanged
             // exemption amount unchanged
-
+                       
             AdjustedCheckValues expectedCompletedOrderValues =
                 new AdjustedCheckValues(expectedSpendAmount, expectedTaxAmount, expectedExemptionAmount);
 
+            // Arrange
```

Comment:
I don't think this `//Arrange` is necessary since there is one on line 61.

Reply:
True... Some of these tests were weird since there are two parts to the test - proposed / completed